<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>QAddPlugin</class>
 <widget class="QWidget" name="QAddPlugin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>565</width>
    <height>224</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>添加项目</string>
  </property>
  <widget class="QWidget" name="layoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>540</width>
     <height>208</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="3">
       <widget class="QPushButton" name="btn_browse">
        <property name="text">
         <string>浏览</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QLabel" name="lab_username">
        <property name="text">
         <string>用户名:</string>
        </property>
       </widget>
      </item>
      <item row="7" column="0" colspan="2">
       <widget class="QPushButton" name="btn_cancel">
        <property name="text">
         <string>取消</string>
        </property>
       </widget>
      </item>
      <item row="1" column="0">
       <widget class="QLabel" name="lab_conf_path">
        <property name="text">
         <string>配置路径:</string>
        </property>
       </widget>
      </item>
      <item row="1" column="1" colspan="2">
       <widget class="QLineEdit" name="txt_config_path">
        <property name="text">
         <string>E:\</string>
        </property>
       </widget>
      </item>
      <item row="4" column="1" colspan="3">
       <widget class="QLineEdit" name="txt_password">
        <property name="text">
         <string>123456</string>
        </property>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="lab_password">
        <property name="text">
         <string>密码:</string>
        </property>
       </widget>
      </item>
      <item row="2" column="1" colspan="3">
       <widget class="QLineEdit" name="txt_ftp">
        <property name="text">
         <string>219.146.3.7</string>
        </property>
       </widget>
      </item>
      <item row="3" column="1" colspan="3">
       <widget class="QLineEdit" name="txt_username">
        <property name="text">
         <string>hx</string>
        </property>
       </widget>
      </item>
      <item row="0" column="1" colspan="3">
       <widget class="QLineEdit" name="txt_name">
        <property name="text">
         <string>悉尼将</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QLabel" name="lab_name">
        <property name="text">
         <string>项目名称:</string>
        </property>
       </widget>
      </item>
      <item row="7" column="2" colspan="2">
       <widget class="QPushButton" name="btn_download">
        <property name="text">
         <string>添加项目</string>
        </property>
       </widget>
      </item>
      <item row="5" column="1" colspan="2">
       <widget class="QCheckBox" name="checkBox">
        <property name="text">
         <string>是否开启网络同步</string>
        </property>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QLabel" name="lab_ftp">
        <property name="text">
         <string>ftp地址:</string>
        </property>
       </widget>
      </item>
      <item row="6" column="0" colspan="4">
       <widget class="QProgressBar" name="progressBar">
        <property name="value">
         <number>0</number>
        </property>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QListWidget" name="list_log"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
